src/Body.jl

```diff
@@ -60,6 +56,13 @@ end
 μ₀(d,ϵ) = kern₀(clamp(d/ϵ,-1,1))
 μ₁(d,ϵ) = ϵ*kern₁(clamp(d/ϵ,-1,1))
 
+"""
+    d = sdf(a::AbstractBody,x,t=0;fastd²=0)
+
+Measure only the distance. Defaults to fastd²=0 for quick evaluation.
+"""
+sdf(body::AbstractBody,x,t=0;fastd²=0,kwargs...) = measure(body,x,t;fastd²)[1]
```

Comment:
When are we using `kwargs` here other than `fastd²`?

Reply:
Nowhere that I can think of...

Reply:
Couldn't find it either. Maybe we should clean it up then!

Reply:
ok. done that.
Do you think I should default to fast or slow evaluation... 0 or Inf?